### ISSUE TITLE
`Error: fork/exec /usr/bin/mysql: inappropriate ioctl for device`

### ISSUE THREAD

**Issue by michaelhays** [planetscale/cli#856]
Hello, I've been running this command to seed my branch databases for a couple of years now:

```
PSCALE_ALLOW_NONINTERACTIVE_SHELL=true pscale shell {database} {branch} < ./path/to/my/data.sql
```

With `pscale` 0.196.0, I'm getting the following error:

```
Error: fork/exec /usr/bin/mysql: inappropriate ioctl for device
```

Version info: `pscale version 0.196.0 (build date: 2024-04-22T13:09:35Z commit: 823251d)`

**Comment by dbussink** [comment]
I think that https://github.com/planetscale/cli/pull/857 should fix this issue.

**Comment by dbussink** [comment]
Released in 0.197.0.

**Comment by michaelhays** [comment]
Thanks @dbussink! 0.197.0 is working for me both on my local computer (Linux, Fedora 39), and GitHub Actions (Linux).
